plugins/inputs/turbostat/plugin.go

```diff
@@ -0,0 +1,87 @@
+package turbostat
```

Comment:
Typically, the main struct of the plugin and its primary functions are placed in a file named after the plugin itself (in this case, it would be `turbostat.go`). 
Generally, I think these files are small enough that their content can be merged.

Reply:
Done.